zeebe/protocol/src/main/resources/cluster-management-protocol.xml

```diff
@@ -10,8 +10,9 @@
     <enum name="AdminRequestType" encodingType="uint8">
       <validValue name="STEP_DOWN_IF_NOT_PRIMARY">0</validValue>
       <validValue name="PAUSE_EXPORTING">1</validValue>
-      <validValue name="RESUME_EXPORTING">2</validValue>
-      <validValue name="BAN_INSTANCE">3</validValue>
+      <validValue name="SOFT_PAUSE_EXPORTING">2</validValue>
+      <validValue name="RESUME_EXPORTING">3</validValue>
+      <validValue name="BAN_INSTANCE">4</validValue>
```

Comment:
:x: This is breaking change. You cannot change the order of existing enums in the protocol. You have to add `SOFT_PAUSE_EXPORTING` at the end.

Reply:
:thinking: Not sure why this was not caught by the backward compatibility checker?